tests/unit/regression/krrnystrom_unittest.cc

```diff
@@ -0,0 +1,186 @@
+/*
```

Comment:
Usually, our naming scheme is a bit different, see the other files.

Minor ...